src/core/manager.c

```diff
@@ -1894,7 +1897,8 @@ static void manager_preset_all(Manager *m) {
         UnitFilePresetMode mode =
                 ENABLE_FIRST_BOOT_FULL_PRESET ? UNIT_FILE_PRESET_FULL : UNIT_FILE_PRESET_ENABLE_ONLY;
 
-        r = unit_file_preset_all(RUNTIME_SCOPE_SYSTEM, 0, NULL, mode, NULL, NULL);
+        r = unit_file_preset_all(RUNTIME_SCOPE_SYSTEM, 0, NULL, mode, &changes, &n_changes);
```

Comment:
Maybe better to log something before calling unit_file_preset_all()? E.g.
```c
log_info("Applying preset policy.");
r = unit_file_preset_all(...);
```

Reply:
applied